### ISSUE TITLE
bootstrap.sh doesn't install bower

### ISSUE THREAD

**Issue by axman6** [damncabbage/ylj16-workshop#2]
Running `./bootstrap.sh`, I'm getting the following output on OS X El Capitan:

```
./bootstrap.sh                                                                               [7h56m]
*** Node.js v5.10.0 already installed.
*** npm 3.8.3 already installed.
*** I'm just about to install purescript@0.8.5 pulp purescript-psa and pscid globally with npm install -g - is that okay? ([y]es, [n]o or [s]kip) > y
*** Proceeding...
/usr/local/bin/pscid -> /usr/local/lib/node_modules/pscid/index.js
/usr/local/bin/pulp -> /usr/local/lib/node_modules/pulp/index.js
/usr/local/bin/psc-docs -> /usr/local/lib/node_modules/purescript/bin/psc-docs.js
/usr/local/bin/psc-bundle -> /usr/local/lib/node_modules/purescript/bin/psc-bundle.js
/usr/local/bin/psc-ide-client -> /usr/local/lib/node_modules/purescript/bin/psc-ide-client.js
/usr/local/bin/psc -> /usr/local/lib/node_modules/purescript/bin/psc.js
/usr/local/bin/psc-ide-server -> /usr/local/lib/node_modules/purescript/bin/psc-ide-server.js
/usr/local/bin/psc-publish -> /usr/local/lib/node_modules/purescript/bin/psc-publish.js
/usr/local/bin/psci -> /usr/local/lib/node_modules/purescript/bin/psci.js
/usr/local/bin/psa -> /usr/local/lib/node_modules/purescript-psa/index.js

> purescript@0.8.5 postinstall /usr/local/lib/node_modules/purescript
> node --harmony_destructuring lib/install.js

  ✔ psc pre-build test passed successfully
  ✔ psc-bundle pre-build test passed successfully
  ✔ psc-docs pre-build test passed successfully
  ✔ psc-ide-client pre-build test passed successfully
  ✔ psc-ide-server pre-build test passed successfully
  ✔ psc-publish pre-build test passed successfully
  ✔ psci pre-build test passed successfully
/usr/local/lib
├── pscid@1.7.2 
├── pulp@8.1.1 
├── purescript@0.8.5 
└── purescript-psa@0.3.7 

*** I'm just about to bower install, npm install, generate development documentation and attempt builds for every exercise directory - is that okay? ([y]es, [n]o or [s]kip) > y
*** Proceeding...
+++ ./01 Basics/01 Building Things, Finding Things, Trying Things/Exercise +++
sh: bower: command not found
+++ ./01 Basics/02 Here's One We Messed Up Earlier/Exercise +++
sh: bower: command not found
+++ ./01 Basics/03 Types and Patterns/Exercise +++
sh: bower: command not found
+++ ./01 Basics/04 Maybe it's Just Nothing/Exercise +++
sh: bower: command not found
+++ ./02 Rows and Effects/01 Places to Be/Exercise +++
sh: bower: command not found
+++ ./02 Rows and Effects/02 Things to Do/Exercise +++
sh: bower: command not found
+++ ./02 Rows and Effects/03 Composing Effects/Exercise +++
sh: bower: command not found
+++ ./03 Web with Pux/01 States and Views/Exercise +++
sh: bower: command not found
+++ ./03 Web with Pux/02 Actions and Updates/Exercise +++
sh: bower: command not found
+++ ./03 Web with Pux/03 Effectful Updates/Exercise +++
sh: bower: command not found
+++ ./03 Web with Pux/04 Skynet/Exercise +++
sh: bower: command not found
+++ ./04 FFI Playground/Exercise +++
sh: bower: command not found
+++ ./01 Basics/02 Here's One We Messed Up Earlier/Answer +++
sh: bower: command not found
+++ ./01 Basics/03 Types and Patterns/Answer +++
sh: bower: command not found
+++ ./01 Basics/04 Maybe it's Just Nothing/Answer +++
sh: bower: command not found
+++ ./02 Rows and Effects/01 Places to Be/Answer +++
sh: bower: command not found
+++ ./02 Rows and Effects/02 Things to Do/Answer +++
sh: bower: command not found
+++ ./02 Rows and Effects/03 Composing Effects/Answer +++
sh: bower: command not found
+++ ./03 Web with Pux/01 States and Views/Answer +++
sh: bower: command not found
+++ ./03 Web with Pux/02 Actions and Updates/Answer +++
sh: bower: command not found
+++ ./03 Web with Pux/03 Effectful Updates/Answer +++
sh: bower: command not found
+++ ./03 Web with Pux/04 Skynet/Answer +++
sh: bower: command not found
```

**Comment by axman6** [comment]
I should note, for anyone playing at home, a simple `npm install -g bower` fixes the problem.

**Comment by damncabbage** [comment]
Fixed by #1 / #3; thanks for the report. :smile:
